CHANGELOG.md

```diff
@@ -10,6 +10,103 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.35.0
+
+### Beta release of the official Nuxt Sentry SDK
+
+This release marks the beta release of the `@sentry/nuxt` Sentry SDK. For details on how to use it, check out the
+[Sentry Nuxt SDK README](https://github.com/getsentry/sentry-javascript/tree/develop/packages/nuxt). Please reach out on
+[GitHub](https://github.com/getsentry/sentry-javascript/issues/new/choose) if you have any feedback or concerns.
+
+- **feat(nuxt): Make dynamic import() wrapping default
+  ([#13958](https://github.com/getsentry/sentry-javascript/pull/13958))** (BREAKING)
+- **feat(nuxt): Add Rollup plugin to wrap server entry with `import()`
+  ([#13945](https://github.com/getsentry/sentry-javascript/pull/13945))**
+
+There is no need to add the Node `--import` flag anymore because the SDK now wraps your application with a dynamic
+`import()`. Please update your start command to avoid initializing Sentry twice (BREAKING CHANGE). If you prefer to use
+the `--import` flag, you can enable this option in your `nuxt.config.ts`:
```

Comment:
```suggestion
**It is no longer required to add a Node `--import` flag. Please update your start command to avoid initializing Sentry twice (BREAKING CHANGE).** The SDK will now apply modifications during the build of your application to allow for patching of libraries during runtime. If run into issues with this change, you can disable this behavior in your `nuxt.config.ts` and use the `--import` flag instead:
```